draft-ietf-acme-acme.md

```diff
@@ -720,18 +736,18 @@ establish a new account with the server, prove control of an identifier, issue a
 certificate, and fetch an updated certificate some time after issuance.  The
 "->" is a mnemonic for a Location header pointing to a created resource.
 
-| Action                | Request                   | Response       |
-|:----------------------|:--------------------------|:---------------|
-| Get directory         | GET  directory            | 200            |
-| Get nonce             | HEAD newNonce             | 200            |
-| Create account        | POST newAccount           | 201 -> account |
-| Submit order          | POST newOrder             | 201 -> order   |
-| Fetch challenges      | GET  order authorizations | 200            |
-| Respond to challenges | POST challenge urls       | 200            |
-| Poll for status       | GET  order                | 200            |
-| Finalize order        | POST order finalize       | 200            |
-| Poll for status       | GET  order                | 200            |
-| Download certificate  | GET  order certificate    | 200            |
+| Action                | Request                    | Response       |
+|:----------------------|:---------------------------|:---------------|
+| Get directory         | GET  directory             | 200            |
+| Get nonce             | HEAD newNonce              | 200            |
+| Create account        | POST newAccount            | 201 -> account |
+| Submit order          | POST newOrder              | 201 -> order   |
+| Fetch challenges      | POST-as-GET authorizations | 200            |
+| Respond to challenges | POST challenge urls        | 200            |
+| Poll for status       | POST-as-GET order          | 200            |
+| Finalize order        | POST order finalize        | 200            |
+| Poll for status       | POST-as-GET order          | 200            |
+| Download certificate  | POST order certificate     | 200            |
```

Comment:
I'm feeling particularly dense today (jet lag), but why is this POST and not POST-as-GET?